components/infobox/commons/infobox_league.lua

```diff
@@ -491,7 +497,11 @@ function League:_setLpdbData(args, links)
 		icon = self.data.icon,
 		icondark = self.data.iconDark,
 		series = mw.ext.TeamLiquidIntegration.resolve_redirect(args.series or ''),
-		seriespage = mw.ext.TeamLiquidIntegration.resolve_redirect(args.series or ''):gsub(' ', '_'),
+		seriespage = pageifyLink(mw.ext.TeamLiquidIntegration.resolve_redirect(args.series or '')),
+		serieslist = {
+			pageifyLink(mw.ext.TeamLiquidIntegration.resolve_redirect(args.series or '')),
+			pageifyLink(mw.ext.TeamLiquidIntegration.resolve_redirect(args.series2 or '')),
+		},
```

Comment:
what if there are >2 series?
if we want to support that could use something like:
```lua
		seriesList = Array.map(Array.map(Array.extractValues(Table.filterByKey(args, function(key)
			return string.find('^series%d*$' ) ~= nil
		end)), mw.ext.TeamLiquidIntegration.resolve_redirect), pageifyLink),
```
(untested)

Reply:
Only display 2 and handle 2 for other purposes now. It cna be changed later, but pairsByPrefix (getAllForBase) should be used over this implementation though